test/services/upload-manager.spec.ts

```diff
@@ -245,6 +245,38 @@ describe('upload-manager.ts', () => {
     expect(chtApi.createUser.callCount).to.be.gt(2); // retried
     expect(chu.uploadError).to.include('could not create user');  
   });
+
+  it('mock data is properly sent to chtApi (multiple roles)', async () => {
```

Comment:
would love to see a targeted test in `place.spec.ts` also targeting the serialization of form data into the place object